IPython/terminal/ptutils.py

```diff
@@ -1,8 +1,18 @@
+"""prompt-toolkit utilities
+
+Everything in this module is a private API,
+not to be used outside IPython.
```

Comment:
Let's add it to the packages skipped for the API docs, in the list [here](https://github.com/ipython/ipython/blob/345bf7d28a831088209162eb855385b94187a362/docs/autogen_api.py#L33).